src/components/Header/AdminHeader/AdminHeader.module.scss

```
@@ -8,18 +8,18 @@
   box-sizing: border-box;
   box-shadow: 2px 0 5px rgb(0 0 0 / 10%);
 
-  .admin-header__title {
+  .title {
```

나중에 contents 안에서 title이라는 명칭을 사용할 수도 있지 않을까 해서 headerTitle은 어떠신지 여쭤봅니당
```suggestion
  .headerTitle {
```